tools/vscode/src/inspect/props.ts

```diff
@@ -182,18 +182,16 @@ export function inspectBinPath(): AbsolutePath | null {
   }
 }
 
-export function inspectLastEvalPath(): AbsolutePath | null {
+export function inspectLastEvalPaths(): AbsolutePath[]  {
   const descriptor = inspectVersionDescriptor();
   const fileName =
     descriptor && descriptor.version.compare(kInspectChangeEvalSignalVersion) < 0
       ? "last-eval"
       : "last-eval-result";
-  const lastEvalFile = join(
-    userRuntimeDir(kPythonPackageName),
-    "view",
-    fileName
-  );
-  return toAbsolutePath(lastEvalFile);
+  
+  return [userRuntimeDir(kPythonPackageName), userDataDir(kPythonPackageName)]
```

Comment:
I'm not sure exactly what's calling what, but could this call have the same issue as the original issue? (When it attempts to resolve `userRuntimeDir`.)

If I'm understanding right, we've so far never hit this codepath in a context where the directory isn't allowed, because we never get as far as opening the viewer.

Reply:
(Then again why would we run the viewer in such a context? Bear in mind the above comment could be nonsense as I'm making several assumptions about how things work.)